addons/mail/static/src/webclient/commands/mail_providers.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+// TODO LEGACY ENV
+import env from "web.env";
+
+const { Component } = owl;
+const { xml } = owl.tags;
+
+class DialogCommand extends Component {}
+DialogCommand.template = xml`
+    <t t-name="web.dialogCommand" owl="1">
+        <div class="o_command_default o_command_dialog">
+            <span t-esc="props.name" />
+            <span t-if="props.email" t-esc="props.email" />
+        </div>
+    </t>`;
```

Comment:
guideline: closing tick of the xml preferably on the same indent level as its opening tick